### ISSUE TITLE
file (/var/www/drupal8dev_install.sh) is absent, cannot continue

### ISSUE THREAD

**Issue by danbohea** [hashbangcode/vlad#143]
I'm seeing this error on `vagrant up` currently:

```
TASK: [drupal | drupal install | set drush/Drupal 8 installation script to executable] ***
failed: [vlad] => {"failed": true, "path": "/var/www/drupal8dev_install.sh", "state": "absent"}
msg: file (/var/www/drupal8dev_install.sh) is absent, cannot continue

FATAL: all hosts have already failed -- aborting
```

**Comment by danbohea** [comment]
This should now be fixed in dev.

Might be an idea to get this merged into master ASAP?

**Comment by philipnorton42** [comment]
Merged with master and tagged version 1.0.1

**Comment by philipnorton42** [comment]
New version (1.0.1) now live.
